.gitattributes

```diff
@@ -0,0 +1,25 @@
+.codacy.yml export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+.travis.yml export-ignore
+Dockerfile export-ignore
+Jenkinsfile export-ignore
+Jenkinsfile.performance export-ignore
+Jenkinsfile.security export-ignore
+LICENSE export-ignore
+NOTICE export-ignore
+README.md export-ignore
+TestDockerfile export-ignore
+bin export-ignore
```

Comment:
bin has python files we should probably scan

Reply:
Added.